docs/entity-cheat-sheet.rst

```diff
@@ -0,0 +1,333 @@
+.. spelling:word-list::
+    hh
+    num
+
+.. comment
+   add an og image like this :og:image: https://docs.getodk.org/_static/img/tutorial-first-form.png
+
+Entity Cheat Sheet
+=================================
+
+This page provides a quick reference for how to design forms in :doc:`XLSForm </tutorial-first-form/>` for ODK that use, create, and update Entities.
+
+Creating and Updating Entity Data
+---------------------------------
+
+When you are designing your form, you can specify how transform submissions into entities. (Learn more about entities :doc:`here </entities-intro/>`.)
+
+To make a form that creates or updates entities, there are two places to make changes in your form: the ``survey`` sheet and the ``entities`` sheet.
+
+
+Survey Sheet
+____________
+
+* Add a ``save_to`` column. 
+  
+  * We recommend putting the ``save_to`` column to the right of the ``required`` column (instead of at the end) so it is easier to see.
+
+* For each field you want to save to an entity, fill out the corresponding entity *property name* in this column. 
+  
+  * The property name can be different from the field name.
+  * Not every field needs to be saved to a property. Only fill out the fields you need for your entity workflow.
+
+
+.. note::
+   Entity property names follow the same naming rules as form field names. Additionally, the property names ``name``, ``label``, and anything beginning with ``__`` (two underscores) are not allowed. 
+
+
+.. list-table:: Example ``survey`` sheet with ``save_to`` column filled in with property names for certain fields 
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - ``type``
+     - ``name``
+     - ``label``
+     - ...
+     - ``save_to``
+   * - geopoint
+     - location
+     - Tree Location
+     - ...
+     - location
+   * - text
+     - species
+     - Tree Species
+     - ...
+     - species
+   * - text
+     - intake_notes
+     - Intake Notes
+     - ...
+     - 
+
+Entities Sheet
+______________
+
+The ``entities`` sheet is included in the XLSForm template, but you can add it yourself if your form does not have one. The required column headers include ``list_name`` and ``label`` but there are several optional column headers depending on your desired functionality.
```

Comment:
Link to XLSForm template

Reply:
Should I link to the google one? or the tutorial that has a link to the google sheets template and the tutorial? I linked to the tutorial above because of this. But I could also just link to the google one so it's one less step.